packages/app/src/app/overmind/effects/vscode/index.ts

```diff
@@ -595,40 +595,6 @@ export class VSCodeEffect {
     }
   };
 
-  public async openDiff(sandboxId: string, module: Module, oldCode: string) {
```

Comment:
I namesearched `openDiff` and this didn't seem to be used anywhere - hopefully I removed the code in this file appropriately. Not sure about the filesystem stuff below.

Reply:
Ah, this part needs to go back. This is not related to offline recover, this is related to GitHub functionality and resolving conflicts. we just use the term "recover" here so deal with the conflict :)